### DISCUSSION TITLE
Current values of variables in channel settings.

### DISCUSSION THREAD

**Discussion by stikstone** [Netgalleria/arska-node#34] [Ideas]
Hello. I will put this here, since it is for sure not an important thing, but in my opinion could be great for setup and checking.

<img width="457" alt="image" src="https://github.com/Netgalleria/arska-node/assets/119031798/9eee62a2-ad42-4592-92c4-c593abf5646b">
in this tab, woukd be In my opinion nice to see the current value of selected variable, why not also the logical answer. Would save some time and effort when there would be less need to scroll between tables when setting up something more complex.

**Comment by Olli69** [comment]
Thanks @stikstone ,
Good to have comments on the UI side also! That is a good idea and would make it more informative at least in some user cases - On the other rule handling UI already quite complex and I try not to touch it, if not necessary 🤓- but maybe in the future  .

Basically adding  current value (when the page is loaded) would be quite easy - dynamic updating values would make it more complicated. Values would take some space in the UI- maybe they could be optional or tooltips ...

Logical answer would be in that case also nice to have, but a bit harder to implement. Logic is now on the MCU and it should probably be cloned to the UI part.

In the short term to make rules more understandable, I have added same explanation texts to the rule templates. It is not the same thing, but example template should makea bit easier for users to start define own rules.
